vim/yanking-text-to-register.md

```diff
@@ -0,0 +1,21 @@
+# Yanking text to register
+
+Yanking and deleting text will save the text to the same unnamed register. Which means every time you delete a word, your 
+yanked word will be overwritten. Which caused me endless amounts of frustration when I accidentally deleted a word and had to
+remember what I yanked earlier.
+
+I learned that you can specify a register to save your yanked word to by prepending your yank command with `"a`. Where `a`
+is a register. There are a number of different registers which do different things. If you want to learn more read vim's [register
+documentation](http://vimdoc.sourceforge.net/htmldoc/change.html#registers).
+
+An example usage is:
+
+`"ayw`
+
+This will save the yanked word to the register a. After deleting various things in your file you can always paste your yanked
```

Comment:
Ah missed that one. Thanks for picking that up. Fixed in commit 4978896c50842e90dd6e546989ce91da16201049.